examples/next/docs/js/settings/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "vanilla-settings-example-testwip",
+  "private": true,
+  "version": "8.1.0",
+  "description": "Javascript settings example",
+  "main": "index.html",
+  "scripts": {
+    "preinstall": "node scripts/removeNodeModules.js && node scripts/test-config.js",
```

Comment:
Can we use `rimraf` to remove `node_modules`? IMO no need for a custom script. We should have the tool already in devDeps

Reply:
I've refactored this code and removed this script as it's no longer needed https://github.com/handsontable/handsontable/pull/7515/commits/984c0dc2555bede7d4309ff5f7bf4e01c9b0e0ef. Maybe create script that will remove `node_modules` for each code example in `examples` directory? It might be helpful for local development

Reply:
@jansiegel mentioned somewhere that `npm ci` will always remove `node_modules` and use `package-lock`. Maybe we should use it instead `npm install` to keep the examples clean and the tests results repeatable.

Reply:
But `npm ci` won't update `package-lock.json`. So when new examples are made, the `package-lock.json` will be outdated.

Reply:
Well... for versioned examples (`/8.3.0` dir) this should be a good thing. We don't want those examples to update the deps and break when they were tested and deployed with particular deps versions.

Dunno about the `next` and when a new version is created scenario. I guess then we would like to have newest version installed and package-lock generated. Maybe if we won't have `package-lock.json` in the `/next` examples it will be generated on `npm run examples:version` and all (and only) the versioned examples will be correctly locked..

I will wait for @jansiegel input on this